scss/bootstrap/components/_wizard.scss

```diff
@@ -45,12 +45,12 @@
         }
 
         &:after {
-          border-left-color: $gray-lighter;
+          border-left-color: $crm-wizard-active-bg-color;
           right: -7px;
         }
 
         &:before {
-          border-left-color: $crm-white;
+          border-left-color: #FFF;
```

Comment:
what's wrong with `$crm-white`?

Reply:
It was done because this variable is not available in SSP. Changed to #ffffff as discussed.